### ISSUE TITLE
[OS X] Ctrl+F2 keyboard shortcut to focus menu bar does not work (accessibility)

### ISSUE THREAD

**Issue by mxdbx** [electron/electron#27268]
### Preflight Checklist

* [ x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [ x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [ x] I have searched the issue tracker for an issue that matches the one I want to file, without success. -> I can see one that was resolved in 2016, but it seems to have regressed.

### Issue Details

* **Electron Version:** 9.0.4
* **Operating System:** Mac OS 10.15.7
* **Last Known Working Electron version:**(unknown) 

### Expected Behavior
Pressing CTRL F2 should focus on the app menu bar (Apple Logo)

### Actual Behavior
Pressing CTRL F2 does nothing.
The only way to focus the app menu bar is to press CTRL F3 (which focuses the Dock), then press CTRL F2. I am not able to get around this in my project and it does not work in any production electron apps (e.g. VSCode/Slack etc...). Native apps do not have an issue with this.

### To Reproduce
CTRL F2 does not focus app menu bar.

This was reported and fixed in 2016 : https://github.com/electron/electron/issues/6016 , but it seems to have regressed.

**Comment by mxdbx** [comment]
Thanks! Is there any plan to backport 9-x-y? It seems that it has only been backported to 10-x-y

**Comment by nornagon** [comment]
9.x is only receiving security updates.
